internal/impl/kafka/integration_test.go

```diff
@@ -821,6 +822,188 @@ output:
 	}
 }
 
+func TestSchemaRegistryProtobufSchemasIntegration(t *testing.T) {
+	integration.CheckSkip(t)
+
+	pool, err := dockertest.NewPool("")
+	require.NoError(t, err)
+	pool.MaxWait = time.Minute
+
+	sr, err := redpandatest.StartRedpanda(t, pool, false, true)
+	require.NoError(t, err)
+
+	t.Logf("Schema Registry URL: %s", sr.SchemaRegistryURL)
+
+	tests := []struct {
+		name       string
+		iterations int
+		normalize  bool
+		metadata   string
+		ruleSet    string
+	}{
+		{
+			name:       "allows creating the same schema twice",
+			iterations: 2,
+		},
+		{
+			name:       "normalises schemas",
+			iterations: 1,
+			normalize:  true,
+		},
+		{
+			name:       "removes metadata",
+			iterations: 1,
+			metadata: `{
+  "properties": {
+    "confluent:version": "1"
+  }
+}`,
```

Comment:
It feels like those should be subtests, not table driven tests.

Reply:
But they are subtests. See below... How should I refactor them?

Reply:
The point is that they are different enough that the table doe not add value i.e. table could be removed.

Reply:
Done. Guess it looks a bit less convoluted than the table-driven version I had before. WDYT?